src/components/avatar/avatar.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import oneLineTrim from 'common-tags/lib/oneLineTrim';
 import { css } from '@emotion/core';
-import vars from '../../../materials/custom-properties';
+import * as vars from '../../../materials/custom-properties';
```

Comment:
This changed as you otherwise would import one token?

Reply:
This was actually not quite correct before, the import was `../../../materials/custom-properties` which implicitly loaded the JSON file.
It should have been `../../../materials/custom-properties.json` to be more correct.

Now it loads the JS version, which contains named exports, hence the `import * as vars`.

Hope it makes sense.